src/service/fixed-layer.js

```diff
@@ -128,6 +128,32 @@ export class FixedLayer {
   }
 
   /**
+   * Apply or reset transform style to fixed elements
+   * @param {?string} transform
+   */
+  transformMutate(transform) {
+    if (transform) {
+      // Apply transform style to all fixed elements
+      this.fixedElements_.forEach(e => {
+        if (e.fixedNow && e.top) {
+          if (e.transform && e.transform != 'none') {
+            setStyle(e.element, 'transform', e.transform + ' ' + transform);
+          } else {
+            setStyle(e.element, 'transform', transform);
+          }
+        }
+      });
+    } else {
+      // Reset transform style to all fixed elements
+      this.fixedElements_.forEach(e => {
+        if (e.fixedNow && e.top) {
+          setStyle(e.element, 'transform', '');
```

Comment:
This looks like it'll trample over the transforms on the element. Should be `e.transform || ''`, right?